internal/grpc/services/gateway/ocmshareprovider.go

```diff
@@ -267,8 +205,8 @@ func (s *svc) UpdateReceivedOCMShare(ctx context.Context, req *ocm.UpdateReceive
 					panic("gateway: error updating a received share: the share is nil")
 				}
 
-				if share.GetShare().ShareType == ocm.Share_SHARE_TYPE_TRANSFER {
```

Comment:
@redblom for your info this is now how an OCM share is identified to be a `datatx` one